api/tests/playbooks/environment_vars.yml

```diff
@@ -29,14 +29,32 @@
         type_name: "{{ environment_ns }}"
         verify_value: true
 
-    - name: Add Lagoon environment variable.
+    - name: Add empty string as Lagoon environment variable.
```

Comment:
This added test could be considered a duplicate of the test that precedes it but it does ensure empty values can be set.